src/main/java/mate/academy/Main.java

```diff
@@ -1,7 +1,25 @@
 package mate.academy;
 
+import java.math.BigDecimal;
+import java.util.List;
+import java.util.Optional;
+import mate.academy.dao.BookDao;
+import mate.academy.lib.Injector;
+import mate.academy.model.Book;
+
 public class Main {
-    public static void main(String[] args) {
+    private static final Injector injector = Injector.getInstance("mate.academy");
 
+    public static void main(String[] args) {
+        BookDao bookDao = (BookDao) injector.getInstance(BookDao.class);
+        Book book = new Book(1L,"Harry Potter and The Chamber of Secrets", new BigDecimal(123));
+        bookDao.create(book);
```

Comment:
don't pass id when you create a book, DB is responsible for handling them, you don't use it, create another constructor or use setters
don't hardcode ids in main, use getId() on saved entity